rmw_connext_shared_cpp/src/node_names.cpp

```diff
@@ -96,27 +106,28 @@ get_node_names(
 
       if (name_found != map.end()) {
         name = std::string(name_found->second.begin(), name_found->second.end());
-      }
-      if (name_found != map.end()) {
```

Comment:
Looks like this was previously a bug. Instead of removing the condition, I think it should be:

```c++
if (ns_found != map.end()) {
```

Reply:
Done